encodings/protobuf/proto/com/github/ocsf/ocsf_schema/v1/ocsf.proto

```diff
@@ -0,0 +1,12805 @@
+// -*-protobuf-*-
+//
+// this is a protobuf representation of the
+// OSF schema 'ocsf' at version 1.2.0
+
+syntax = "proto3";
+package com.github.ocsf.ocsf_schema.v1;
+
+option go_package = "github.com/ocsf/examples/sdk/go/pb";
+
+import "google/protobuf/struct.proto";
+import "google/protobuf/timestamp.proto";
+
+// FileActivity is an OSF event in category "System Activity"
+// called File System Activity
+// defined in ocsf events/system/filesystem.json
+//
+// File System Activity events report when a process performs an action on a
+// file or folder.
+message FileActivity {
+    enum ActionId {
+        ACTION_ID_UNKNOWN = 0;  // The action was unknown. The <code>disposition_id</code>
```

Comment:
The enumerand names here don't match what's described above. (They also use the fixed prefix style I like more too.)

BTW: +1 for use of the term "enumerand"!